File: 백준/Bronze/2920. 음계/음계.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();        
        int flag = 0;
        for (int i = 0; i < 7; i++) {
            int m = sc.nextInt();
            if (m == n + 1) {
                flag = 1;
            } else if (m == n - 1) {
                flag = 2;
            } else {
                flag = 0;
                System.out.println("mixed");
                break;
            }
            n = m;
        }

        if (flag == 1) {
            System.out.println("ascending");
        }
        if (flag == 2) {
            System.out.println("descending");
        }
    }   
}

File: 백준/Bronze/2920. 음계/음계.js
/** @format */
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\r?\n/);

function solution(input) {
    const arr = input[0].split(" ").map(Number);

    let ans = "";

    if (arr[0] < arr[1]) ans = "ascending";
    if (arr[0] > arr[1]) ans = "descending";

    for (let i = 2; i < arr.length; i++) {
        if (ans === "ascending" && arr[i - 1] > arr[i]) {
            console.log("mixed");
            return;
        }

        if (ans === "descending" && arr[i - 1] < arr[i]) {
            console.log("mixed");
            return;
        }
    }

    console.log(ans);
}
solution(input);